src/DotNet.Testcontainers.Tests/Unit/GitHub.cs

```diff
@@ -0,0 +1,17 @@
+namespace DotNet.Testcontainers.Tests.Unit
+{
+  using System;
+  using System.Threading.Tasks;
+  using DotNet.Testcontainers.Clients;
+  using Xunit;
+
+  public class GitHub
+  {
+    [Fact]
+    public Task PullRequest361()
```

Comment:
```suggestion
    public void PullRequest361()
```

Reply:
This was just a test commit to see if it works on Unix. `mapName` is not supported. I will change it to `CreateFromFile` in the next days and test it on a separate branch. Than I'll merge it back to this pr.